Updates to teach in England if you trained outside the UK page (#4749)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

          <li><a href="https://www.gov.uk/guidance/support-for-family-members-of-british-nationals-in-ukraine-and-ukrainian-nationals-in-ukraine-and-the-uk">Ukrainian nationals outside of the UK</a></li>
        </ul>
cta_arrow_link:
  nonukqts:
    link_target: "https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers"
    link_text: "Check the eligibility criteria and apply for QTS in England"
  qtsroutes:
    link_target: "https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk"
    link_text: Find out about the different routes to QTS for non-UK citizens
  teachervisa: 
    link_target: "/non-uk-teachers/visas-for-non-uk-teachers"
    link_text: "Learn more about visas for non-UK teachers"
  sponsjobs:

>     link_target: "https://teaching-vacancies.service.gov.uk"

Reviewer: ```suggestion
    link_target: "https://teaching-vacancies.service.gov.uk/jobs?visa_sponsorship_availability%5B%5D=true"
```

    link_text: "Search for teaching jobs with visa sponsorship"
---

$new-international-funding$

Teaching in England offers exciting opportunities, including:

* a [competitive salary](/life-as-a-teacher/pay-and-benefits/teacher-pay)
* support in your professional development
* varied job roles

You will need English [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach in most schools in England. QTS is a professional teaching qualification that teachers in England gain at the end of their training. 

> You will also need a high standard of written and spoken English to teach in England. 

Author: I'm not sure, it doesn't say on the original either so may depend on individual employers. There is a list of standards for trainees though, so can double check with policy

$ukraine-moving$

## If you do not have qualified teacher status

If you do not have QTS, schools will want to see alternative evidence of your teaching and academic qualifications. 

> You can get a [statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx). This will show how your qualifications compare to English ones. 

Reviewer: Shouldn't we include `ENIC` as well as the full name? It is referred to as ENIC on other pages and also elsewhere in the journey (eg Apply)

> You can get a [statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx). This will show how your qualifications compare to English ones. 

Reviewer 2: On other pages we have slightly different content - To check your qualifications meet the required standard in England you will need a [statement of comparability from the UK European Network of Information Centres (UK ENIC)](https://enic.org.uk/Qualifications/SOC/Default.aspx)

> You can get a [statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx). This will show how your qualifications compare to English ones. 

Author: To check your qualifications meet the required standard in England you will need [a statement of comparability from the UK European Network of Information Centres (UK ENIC)](https://enic.org.uk/Qualifications/SOC/Default.aspx).

> You can get a [statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx). This will show how your qualifications compare to English ones. 

Author: ```suggestion
To check your qualifications meet the required standard in England you will need [a statement of comparability from the UK European Network of Information Centres (UK ENIC)](https://enic.org.uk/Qualifications/SOC/Default.aspx).
```

If you are a qualified teacher from outside the UK, you can work as a teacher in England for up to 4 years without QTS.  After that, you will need QTS to teach in many schools in England. 

> $qtsroutes$

Reviewer 2: I think the 2 green arrows is a bit overwhelming - I think the first one is maybe more useful, and that GOV.UK page does link to the second green arrow link anyway!

> $nonukqts$

Author: ```suggestion
```
